backend/app/api/routes/utils.py

```diff
@@ -29,3 +29,7 @@ def test_email(email_to: EmailStr) -> Message:
 @router.get("/health-check/")
 async def health_check() -> bool:
     return True
+
+
+# Permissions
+MANAGE_ORGANIZATION: str = "manage_organization"
```

Comment:
@ninad-ict 

Rather than hardcoding here, may be we should consider writing a helper function.  That checks for role rather than permission?